evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -674,15 +674,24 @@ contract FluxAggregator is AggregatorInterface, Owned {
   function roundState(address _oracle)
```

Comment:
This isn't really the round's state, it the oracle's state. Nor is this result pertaining to a particular round, it's just the state of the oracle in the contract as it is now. The "roundState" can change without a round changing, like in the case where the contract creator tops up with funds. Maybe rename to `oracleState`?

Reply:
I'll mention this to @se3000 -- he's doing some refactoring related to this method in a separate PR.